### PR TITLE
Sto step palettes v2 doc 2989

### PR DESCRIPTION
**Pull request by douglas-j-bothwell**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* https://harness.atlassian.net/browse/DOC-2989

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/security-testing-orchestration/sto-techref-category/brakeman-scanner-reference.md [hunk 1]
```diff
@@ -4,8 +4,174 @@ description: Repository scans with Brakeman
 sidebar_position: 70
 ---
 
+You can scan your container images using [Brakeman](https://brakemanscanner.org/), free tool for scanning Ruby on Rails applications.
```
Comment (StrawberryPad) [inline docs/security-testing-orchestration/sto-techref-category/brakeman-scanner-reference.md:7]:
```suggestion
You can scan your container images using [Brakeman](https://brakemanscanner.org/), a free tool for scanning Ruby on Rails applications. 
```

File: docs/security-testing-orchestration/use-sto/orchestrate-and-ingest/ingest-sarif-data.txt [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+```mdx-code-block
+import StoSettingFailOnSeverity from '../../sto-techref-category/shared/step_palette/_sto-ref-ui-fail-on-severity.md';
+```
+<StoSettingFailOnSeverity />
+
+
+## YAML pipeline example: ingest CodeQL scan results
+
+The following pipeline illustrates how you can set up a scan using a Run step and then ingest the results using a Custom
```
Comment (StrawberryPad) [inline docs/security-testing-orchestration/use-sto/orchestrate-and-ingest/ingest-sarif-data.txt:64]: Sentence seems to be incomplete?

**Review by StrawberryPad — APPROVED** [review]

File: docs/security-testing-orchestration/sto-techref-category/custom-ingest-reference.md [hunk 3]
```diff
@@ -0,0 +1,564 @@
+---
+title: Custom Ingest settings reference
+description: ingest JSON or SARIF data from external scanners.
```
Comment (deepakbolangady) [inline docs/security-testing-orchestration/sto-techref-category/custom-ingest-reference.md:3]:
```suggestion
description: Ingest JSON or SARIF data from external scanners.
```

File: docs/security-testing-orchestration/sto-techref-category/custom-ingest-reference.md [hunk 4]
```diff
@@ -0,0 +1,564 @@
...
+sidebar_position: 105
+---
+
+You can ingest results from any third-party scanner. Your results need to be in either [SARIF](https://docs.oasis-open.org/sarif/sarif/v2.1.0/sarif-v2.1.0.html) or [Harness Custom JSON](/docs/security-testing-orchestration/use-sto/orchestrate-and-ingest/ingesting-issues-from-other-scanners#json-data-format-reference) format.
+
+For a full description of the workflow, go to [Ingest Results from Custom or Unsupported Scanners](/docs/security-testing-orchestration/use-sto/orchestrate-and-ingest/ingesting-issues-from-other-scanners.md).
```
Comment (deepakbolangady) [inline docs/security-testing-orchestration/sto-techref-category/custom-ingest-reference.md:9]: Convert the referenced topic's title to sentence case? Check globally?

**Review by deepakbolangady — COMMENTED** [review]
Hi, Doug. I added some feedback by reading from my local instance, and I hope I've been able to comment on the right locations. I don't see the option to approve. I see only "Submit review" and "Cancel review" buttons. 

Most of my feedback in the first topic can be applied across, and it should be a one-time thing for you since they are references. I stopped after I learned that it's approved.

**Pull request merged**